client.go

```diff
@@ -498,6 +529,24 @@ func bitsToImportRequest(indexName string, frameName string, slice uint64, bits
 	}
 }
 
+func statusToNodeSlicesForIndex(status *Status, indexName string) map[uint64]*URI {
```

Comment:
i don't mind so much that you're swallowing the error as long as that was your intent. perhaps you could leave a comment that you intentionally ignore the error because they are assumed to be the same.